mobile/src/features/Swap/common/SwapProvider.tsx

```diff
@@ -22,6 +22,22 @@ import {undefinedToken} from './constants'
 import {useNavigateTo} from './navigation'
 import {useGetInputs} from './useGetInputs'
 
+// Request ID tracking to ignore outdated responses
```

Comment:
this can be accomplish in a fashion manner with `rxjs` it will automatically SignAbort previous requests

Reply:
It could, but is it worth it the added complexity of refactoring the whole way the estimate is called to have rxjs?

Reply:
Looking into other ways of dealing with this

Reply:
simplified it a bit, but still not worth making a rxjs stream for this I think